apps/storefront/src/App.tsx

```diff
@@ -191,7 +192,11 @@ export default function App() {
       };
 
       if (!customerId) {
-        const info = await getCurrentCustomerInfo();
+        const info = await getCurrentCustomerInfo().catch((error) => {
+          if (isCompanyError(error)) {
+            gotoPage(`/login?loginFlag=${error.reason}`);
```

Comment:
I changed this so we can remove the duplication but keep the fixed message, rather than the snackbar, this is more accessible and harder to miss.